Makefile

```diff
@@ -10,6 +10,8 @@ phan:
 	$(DC_RUN_PHP) env PHAN_DISABLE_XDEBUG_WARN=1 php ./vendor/bin/phan
 examples: FORCE
 	$(DC_RUN_PHP) php ./examples/AlwaysOnTraceExample.php
+	$(DC_RUN_PHP) php ./examples/AlwaysOffTraceExample.php
+	$(DC_RUN_PHP) php ./examples/JaegerExporterExample.php
```

Comment:
This is great for now - eventually we may want to turn these into integration tests.

Reply:
Also, perhaps we want to perform a `docker-compose up -d` in `make examples` before we execute the examples?  That may take away some of the confusion of running these examples.

Reply:
yes, totally make sense to add `docker-compose up -d` in make examples. I will do it in this PR.

Reply:
We should also probably update the examples section in the README with this change:

https://github.com/open-telemetry/opentelemetry-php/#examples